spot-client/src/common/app-state/calendars/selectors.js

```diff
@@ -1,3 +1,5 @@
+import { getCalendarConfig } from '../config/selectors';
+import { integrationTypes } from '../../../spot-tv/calendars';
```

Comment:
Not sure I understand the problem here. Is it that we're trying to avoid importing Spot TV or Spot Remote into the common ?

Reply:
Yeah. common should be importing from itself and node_modules; it is the dumping ground for logic that can be used by spot-tv and spot-remote while keeping spot-tv and spot-remote separate. spot-remote should be importing from itself and node_modules and common. spot-tv should be importing from itself and node_modules and common.

Reply:
So for example one way to make me shut up would be move the calendar integration types constants into common so spot-remote doesn't indirectly import something from spot-tv.

Reply:
So I moved it to common and then renamed, because it was no longer in "calendar" package. Let me know if that's okay or should I move it to a dedicated "calendar" space in the common.

Reply:
Good enough to me.